[2025-02-11T22:12:53.790+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-11T22:12:53.819+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2025-02-11T22:12:44.934691+00:00 [queued]>
[2025-02-11T22:12:53.833+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2025-02-11T22:12:44.934691+00:00 [queued]>
[2025-02-11T22:12:53.834+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-11T22:12:53.858+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): transform_load_weather_data> on 2025-02-11 22:12:44.934691+00:00
[2025-02-11T22:12:53.870+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=570) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-11T22:12:53.873+0000] {standard_task_runner.py:72} INFO - Started process 572 to run task
[2025-02-11T22:12:53.873+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'transform_load_weather_data', 'manual__2025-02-11T22:12:44.934691+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpfopk6oll']
[2025-02-11T22:12:53.878+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask transform_load_weather_data
[2025-02-11T22:12:53.953+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_dag.transform_load_weather_data manual__2025-02-11T22:12:44.934691+00:00 [running]> on host dbef5e540f20
[2025-02-11T22:12:54.092+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='aida' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='transform_load_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T22:12:44.934691+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-11T22:12:44.934691+00:00'
[2025-02-11T22:12:54.094+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-11T22:12:54.181+0000] {logging_mixin.py:190} INFO -        City Description  ...  Sunrise (Local Time)  Sunset (Local Time)
0  Portland   clear sky  ...   2025-02-11 07:18:10  2025-02-11 17:31:53

[1 rows x 12 columns]
[2025-02-11T22:12:54.187+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-11T22:12:54.206+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-11T22:12:54.208+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=transform_load_weather_data, run_id=manual__2025-02-11T22:12:44.934691+00:00, execution_date=20250211T221244, start_date=20250211T221253, end_date=20250211T221254
[2025-02-11T22:12:54.254+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-11T22:12:54.288+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-11T22:12:54.291+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
